app/src/main/java/org/eyeseetea/malariacare/domain/entity/Question.java

```diff
@@ -335,7 +330,7 @@ public static Builder newBuilder() {
             return conditions;
         }
 
-        public void setConditions(
+        private void setConditions(
```

Comment:
Can you remove private setters? It's not necessary we have private setters, so we reduce the size of the class

Reply:
Done!